### PR TITLE
learntheconcepts --> edits to agent-based and multi-agents systems section

### PR DESCRIPTION
**Pull request by FelixNicolaeBucsa**
…ection

### CODE REVIEW

File: docs/learn_the_concepts/agent-based_and_multi-agents_systems.md [hunk 1]
```diff
@@ -18,46 +18,56 @@ Agents have _five key characteristics_:
...
 
-_Agent characteristics and behaviors may vary in their extent and sophistication_. An agent may differ from other agents, for instance, in the amount of information considered in the agent’s decisions process, its internal models of the external world, its view of the possible reactions of other agents in response to its actions, and the size of the memory of past events the agent retains and uses in making its decisions. Further differentiating factors are the amounts of resources used by agents or accumulated as a result of their interactions.
+???+ note
+
+    Agent characteristics and behaviors may vary in their extent and sophistication. An agent may differ from other agents, for instance, in the amount of information in the agent’s decisions process, its internal models of the external world, its view of the possible reactions of other agents in response to its actions, and the size of the memory of past events the agent retains and uses in making its decisions. Further instances of differentiating factors are the amounts of resources used by agents or accumulated as a result of their interactions.
```
Comment (FelixNicolaeBucsa) [inline docs/learn_the_concepts/agent-based_and_multi-agents_systems.md:26]: We could add a table summarising all of the content above, that is, the features of agents

File: docs/learn_the_concepts/agent-based_and_multi-agents_systems.md [hunk 2]
```diff
@@ -18,46 +18,56 @@ Agents have _five key characteristics_:
...
+    Interactions between agents is an important feature that enables them to use knowledge of other agents and learn more about the environment in a compressed time period. This type of interaction may be _cooperative_ or _competitive_. In a cooperative interaction, agents work with each other towards a common goal. The aim of this interaction is to enable agents to distribute and share their knowledge and use the intelligence and capabilities of each other to solve problems. In a competitive interaction, agents may compete to obtain individual resources and achieve individual goals.
```
Comment (FelixNicolaeBucsa) [inline docs/learn_the_concepts/agent-based_and_multi-agents_systems.md:44]: We could create an infographic with benefits and drawbacks in a simplified way so to reduce text content

File: docs/learn_the_concepts/agent-based_and_multi-agents_systems.md [hunk 3]
```diff
@@ -18,46 +18,56 @@ Agents have _five key characteristics_:
...
+!!! example
+
+    Agent-based modelling finds applicatino into the physical, biological, social, and management sciences. Agent-based models have also been developed in the fields of economics, sociology, anthropology, and cognitive science. Various social phenomena have been investigated using agent-based models that are not easily modeled using other approaches. Here, agent-based models are being used to analyze existing and hypothetical markets. FOr instance, agent-based simulations could be used to model possible futures for a market directed towards space tourism, to analyze how companies represented by agents would compete and offer products to customers in this hypothetical market.
```
Comment (VijaySharma815) [inline docs/learn_the_concepts/agent-based_and_multi-agents_systems.md]: Agent-based modelling finds applicatino into the -> Agent-based modelling finds application into the

Comment (VijaySharma815) [inline docs/learn_the_concepts/agent-based_and_multi-agents_systems.md]: FOr instance, agent-based simulations -> For instance, agent-based simulations

**Pull request merged**